### ISSUE TITLE
Unknown language in `contributes.rails.language`

### ISSUE THREAD

**Issue by croko** [bung87/vscode-rails#10]
- Extension Name: rails
- Extension Version: 0.8.1
- OS Version: Darwin x64 17.7.0
- VSCode version: 1.28.0-insider

 We have written the needed data into your clipboard. Please paste:
```json 
{
	"messages": [
		{
			"type": 3,
			"message": "Unknown language in `contributes.rails.language`. Provided value: slim",
			"extensionId": "bung87.rails",
			"extensionPointId": "snippets"
		}
	],
	"activationTimes": {
		"startup": true,
		"codeLoadingTime": 314,
		"activateCallTime": 0,
		"activateResolvedTime": 0,
		"activationEvent": "workspaceContains:**/Gemfile"
	}
}
```

**Comment by bung87** [comment]
well, install slim extension on market for now.

**Comment by bung87** [comment]
fixed 0.8.2
